### PR TITLE
submit a dynamic data source example

### PR DESCRIPTION
**Pull request by a364176773**

### CODE REVIEW

**Comment by a364176773** [comment]
@xingfudeshi  代码内初始化了两遍seata现已移除,麻烦跑一下谢谢

File: dubbo-multiple-datasource-mybatis-plus/consumer/src/main/java/io/seata/samples/config/SeataAutoConfig.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+    /**
+     * init global transaction scanner
+     *
+     * @Return: GlobalTransactionScanner
+     */
+    @Bean
+    public GlobalTransactionScanner globalTransactionScanner() {
+        logger.info("配置seata........");
+        return new GlobalTransactionScanner("test-consumer", "test-consumer");
```
Comment (xingfudeshi) [inline dubbo-multiple-datasource-mybatis-plus/consumer/src/main/java/io/seata/samples/config/SeataAutoConfig.java]: use my_test_tx_group for second parameter.

File: dubbo-multiple-datasource-mybatis-plus/provider/src/main/java/io/seata/samples/config/SeataAutoConfig.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+    /**
+     * init global transaction scanner
+     *
+     * @Return: GlobalTransactionScanner
+     */
+    @Bean
+    public GlobalTransactionScanner globalTransactionScanner() {
+        logger.info("配置seata........");
+        return new GlobalTransactionScanner("test-provider", "test-provider");
```
Comment (xingfudeshi) [inline dubbo-multiple-datasource-mybatis-plus/provider/src/main/java/io/seata/samples/config/SeataAutoConfig.java]: use my_test_tx_group for second parameter.

File: dubbo-multiple-datasource-mybatis-plus/consumer/src/main/java/io/seata/samples/controller/TestController.java [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        return productService.page(new Page<Product>(1, 10));
+    }
+
+    /**
+     * 测试事务提交
+     * 
+     * @return
+     * @throws TransactionException
+     */
+    @GetMapping(value = "testSeataTwo")
```
Comment (xingfudeshi) [inline dubbo-multiple-datasource-mybatis-plus/consumer/src/main/java/io/seata/samples/controller/TestController.java]: Please use testCommit or commit to clarify the purpose of the request URI.

File: dubbo-multiple-datasource-mybatis-plus/consumer/src/main/java/io/seata/samples/controller/TestController.java [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+        return demoService.Two();
+    }
+
+    /**
+     * 测试事务回滚
+     * 
+     * @return
+     * @throws TransactionException
+     */
+    @GetMapping(value = "testSeataOne")
```
Comment (xingfudeshi) [inline dubbo-multiple-datasource-mybatis-plus/consumer/src/main/java/io/seata/samples/controller/TestController.java]: Please use testRollback or rollback to clarify the purpose of the request URI.

File: dubbo-multiple-datasource-mybatis-plus/README.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+根据sql文件夹内的sql文件名创建对应表，运行sql
+
+### 安装Seata跟Nacos
+
+按照该篇[文章](http://seata.io/zh-cn/blog/seata-nacos-analysis.html)安装seata跟nacos配置
+
+## 测试
+
+- 先后启动provider跟comsumer
```
Comment (xingfudeshi) [inline dubbo-multiple-datasource-mybatis-plus/README.md]: It's better to mention all URIs for testing here.
like
http://127.0.0.1:28888/test/pageByProduct
...

Comment (a364176773) [reply dubbo-multiple-datasource-mybatis-plus/README.md]: 已修改,麻烦看下是否有问题,谢谢

**Pull request merged**